tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "esModuleInterop": true,
     "isolatedModules": true,
     "jsx": "react",
-    "module": "commonjs",
+    "module": "ES2022",
+    "moduleResolution": "Bundler",
```

Comment:
This tells typescript we are using a bundler (i.e. webpack), and so to use the module resolution conventions typical of bundlers.